doc/concepts/data-concepts/repo.md

```diff
@@ -0,0 +1,70 @@
+# Repository
+
+A Pachyderm repository is a location where you store your data inside
+Pachyderm. A Pachyderm repository is a top-level data object that contains
+files and folders. Similar to Git, a Pachyderm repository tracks all
+changes to the data and creates a history of data modifications that you
+can access and review. You can store any type of file is in a Pachyderm repo,
+including binary and plain text files.
+
+Unlike a Git repository that stores history in a `.git` file in your copy
+of a Git repo, Pachyderm stores the history of your commits in a centralized
+location. Because of that, you do not run into
+merge conflicts as you often do with Git commits when you try to merge
+your `.git` history with the master copy of the repo. With large datasets
+resolving a merge conflict might not be possible.
+
+A Pachyderm repository is the first entity that you configure when you
+want to add data to Pachyderm. You can create a repository by running
+the `pachctl create repo` command or by using the Pachyderm UI. After
+creating the repository, you can add your data by using the
+`pachctl put file` command.
+
+The following types of repositories exist in Pachyderm:
+
+Input repositories
+ Users or external applications outside of Pachyderm can add data to
+ the input repositories for further processing.
+
+Output repositories
+ Pachyderm automatically creates output repositories
+ pipelines write results of computations into these repositories.
+ The output repository is the `pfs/out` located in your container.
```

Comment:
This can be phrased better:

```suggestion
Any data written to the `pfs/out` directory within your pipeline user container (see pipelines LINK) will be written to that pipeline's output repository.
```